ingest/importer/conversion/template_manager.py

```diff
@@ -165,14 +165,22 @@ def __init__(self, domain_type, object_type, cell_conversions, default_values={}
         self.default_values = copy.deepcopy(default_values)
 
     def do_import(self, row: IngestRow):
-        metadata = MetadataEntity(domain_type=self.domain_type, concrete_type=self.concrete_type,
-                                  content=self.default_values, row=row)
-        for index, cell in enumerate(row.values):
-            if cell.value is None:
-                continue
-            conversion: CellConversion = self.cell_conversions[index]
-            conversion.apply(metadata, cell.value)
-        return metadata
+        row_errors = []
+        metadata = None
+        try:
+            metadata = MetadataEntity(domain_type=self.domain_type, concrete_type=self.concrete_type,
+                                      content=self.default_values, row=row)
+            for index, cell in enumerate(row.values):
+                if cell.value is None:
+                    continue
+                try:
+                    conversion: CellConversion = self.cell_conversions[index]
+                    conversion.apply(metadata, cell.value)
+                except Exception as e:
+                    row_errors.append({"location": f'cell={index}, value={cell.value}', "type": e.__class__.__name__, "detail": str(e)})
```

Comment:
Could return the actual indices for row, column as well, then all the string formatting could happen in the end. It would be handy if we change the format of the error message.

Reply:
This happens as the error bubbles up.
The row and cell importers don't know what worksheet they are on, so this is added by the worksheet: importer.py line 238

Reply:
Oh I meant for the location, the location is a string atm, could just return the actual column index, value. And yes the row and worksheet title will get added later.